packs/data/pfs-season-2-bestiary.db/urxehls-firestorm-7-8.json

```diff
@@ -0,0 +1,210 @@
+{
+    "_id": "AkKxh2OXfWEb5OHt",
+    "data": {
+        "attributes": {
+            "ac": {
+                "value": 0
+            },
+            "hardness": 0,
+            "hp": {
+                "details": "",
+                "max": 0,
+                "temp": 0,
+                "value": 0
+            },
+            "stealth": {
+                "details": "<p>(expert)</p>",
+                "value": 15
+            }
+        },
+        "creatureType": "",
+        "customModifiers": {},
+        "details": {
+            "description": "<p>Abyssal fire rains from the sky.</p>",
+            "disable": "<p>@Check[type:arcana|dc:28], @Check[type:nature|dc:28], @Check[type:occultism|dc:28], or  @Check[type:religion|dc:28] (trained) twice or @Compendium[pf2e.spells-srd.Dispel Magic]{Dispel Magic} (3rd level; counteract DC 22) to cause a temporary, local disruption to the Disciple of Urxehl's ritual. Checks to disrupt the firestorm require two actions. Defeating the Disciple of Urxehl also disables the hazard.</span></p>",
+            "isComplex": true,
+            "level": {
+                "value": 6
+            },
+            "reset": "<p>The hazard activates when the PCs reach Event: Rain of Fire, area A1, and area A9. If the PCs defeat the Disciple of Urxehl, the hazard is permanently destroyed.</p>",
+            "routine": ""
+        },
+        "saves": {
+            "fortitude": {
+                "saveDetail": "",
+                "value": 0
+            },
+            "reflex": {
+                "saveDetail": "",
+                "value": 0
+            },
+            "will": {
+                "saveDetail": "",
+                "value": 0
+            }
+        },
+        "source": {
+            "value": "Pathfinder Society Scenario #2-22: Breaking the Storm: Excising Ruination"
+        },
+        "statusEffects": [],
+        "traits": {
+            "di": {
+                "custom": "",
+                "selected": {},
+                "value": []
+            },
+            "dr": [],
+            "dv": [],
+            "rarity": "unique",
```

Comment:
```suggestion
            "rarity": "rare",
```

Reply:
I did intentionally change this to unique because rare makes zero sense... But I guess it is what the PDF says